spec.bs

```diff
@@ -245,6 +245,77 @@ Run the [=update document frozenness steps=] given <var ignore>child document</v
 
 Each {{HTMLMediaElement}} has a <dfn for="HTMLMediaElement">resume frozen flag</dfn>, which is initially set to false.
 
+Modifications to the Service Worker Standard {#serviceworker-mod}
+--------------------------------------------
+
+### Service Worker: <a href="https://w3c.github.io/ServiceWorker/#client-interface">Client Interface</a> ### {#serviceworker-client-dfn}
+
+<pre class="idl">
+    partial interface Client {
+        readonly attribute  ClientState state;
+    };
+
+    enum ClientState {
+        "active",
+        "frozen"
+    };
+</pre>
+
+A {{Client}} object has an associated <dfn id="dfn-service-worker-client-state" for="Client">state</dfn>, which is one of {{ClientState}} attribute value.
```

Comment:
Should this be for all clients, or just WindowClients?

Reply:
That's how I read it, but it's surprising to me because I thought only pages (Documents) had frozenness.

Reply:
So we might need to adjust the text there because workers become "blocked..."
https://wicg.github.io/page-lifecycle/spec.html#worklets-workers

Reply:
Moving this out of inline comments into the main PR discussion.